src/jaguar.toit

```diff
@@ -26,7 +26,10 @@ RUN_DEFINES_HEADER ::= "X-Jaguar-Run-Defines"
 
 logger ::= log.Logger log.INFO_LEVEL log.DefaultTarget --name="jaguar"
 validate_firmware / bool := firmware.is_validation_pending
-disabled / monitor.Signal? := null
+
+disabled / bool := false
+network_down / monitor.Semaphore ::= monitor.Semaphore
```

Comment:
Would it make sense to call this `network_released`?

Reply:
Went with `network_free`.